static/app/views/performance/cache/referrers.ts

```diff
@@ -0,0 +1,12 @@
+const referrerPrefix = 'api.performance.cache';
+
+enum ReferrerSuffix {
+  LANDING_CACHE_HIT_MISS_CHART = `landing-cache-hit-miss-chart`,
+  LANDING_CACHE_THROUGHPUT_CHART = `landing-cache-throughput-chart`,
+}
+
+type PrefixedReferrer = {
+  [key in keyof typeof ReferrerSuffix]: `${typeof referrerPrefix}.${(typeof ReferrerSuffix)[key]}`;
+};
+
+export const Referrer: PrefixedReferrer = ReferrerSuffix as unknown as PrefixedReferrer;
```

Comment:
@gggritso wdyt if this, no need to hardcode a prefix to the enums, less code to write, less chance of mistakes. I can do the same with http module

Reply:
I'm referring to this file btw https://github.com/getsentry/sentry/blob/master/static/app/views/performance/http/referrers.tsx

Also, super nit, it can probably be a `.ts` file instead

Reply:
Yeah fair enough, with copy and paste also, i doubt this will be a problem anyways, it was neat if anything!